fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -27,6 +29,17 @@ class MirabufSceneObject extends SceneObject {
     private _debugBodies: Map<string, RnDebugMeshes> | null
     private _physicsLayerReserve: LayerReserve | undefined = undefined
 
+    private _transformGizmos: TransformGizmos | undefined = undefined
+    private _deleteGizmoOnEscape: boolean = true
```

Comment:
I'll let this be for now, but the entirety of transform gizmos needs a major refactor to be more adaptable. It should honestly be worked into some kind of UI component. [Jira ticket for refactoring](https://jira.autodesk.com/browse/AARD-1744)